app/assets/javascripts/flood_risk_engine/application.js

```diff
@@ -12,3 +12,11 @@
 //
 //= details.polyfill
 //= require_tree .
+
+
+$(function() {
```

Comment:
ok. Yes I've noticed odd behaviour on most pages if you start using Back and back  link